amundsen_application/static/js/components/common/BadgeList/index.tsx

```diff
@@ -8,14 +8,17 @@ import { getBadgeConfig } from 'config/config-utils';
 import { Badge } from 'interfaces/Tags';
 
 export interface BadgeListProps {
-  badges: Badge[];
+  badges: any[]; // TODO replace with new badges later
 }
 
 const BadgeList: React.FC<BadgeListProps> = ({ badges }: BadgeListProps) => {
   return (
     <span className="badge-list">
       {badges.map((badge, index) => {
-        const badgeConfig = getBadgeConfig(badge.tag_name);
+        var name;
```

Comment:
`let name;`. 

`var` is an old way of declaring all variables in JavaScript. Several years ago a new standard was introduced to use `let` for variables that would be reassignable, and `const` for ones that weren't. Long story short this allowed web engines to optimize certain processes.t.